packages/flutter_tools/templates/plugin_ffi/src.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,23 @@
+# We use the CMake version connected to the version of the Android Gradle Plugin.
```

Comment:
Per previous-cycle review comments, I think "we" is confusing in a template. I would say "This is" rather than "we use"